python/packages/aws-lambda-sdk/pyproject.toml

```diff
@@ -33,6 +35,7 @@ tests = [
     "black>=22.12",
     "boto3>=1.16.112",
     "flask>=2.2.3",
+    "importlib_metadata>=5.2", # included in Python >=3.8
```

Comment:
Why do we need to list it as optional dependency?

Reply:
The test still uses it to infer the version, it will only be installed for the tests and not included in the package that get's distributed.